src/xcode/ENA/ENA/Source/Scenes/Events/Store/Model/Event.swift

```diff
@@ -0,0 +1,22 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+enum EventType: Int {
+	case type1
+	case type2
+	case type3
+}
+
+struct Event {
+	// The ID of the event. Note that the ID is generated by the CWA server. It is stored as base64-encoded string of the guid attribute of Protocol Buffer message Event.
+	let id: String
+	let type: EventType
+	let description: String
+	let address: String
+	let start: Date
+	let end: Date
+	let defaultCheckInLengthInMinutes: Int
+	// The signature of the event (provided by the CWA server). It is stored as a base64-encoded string of the signature attribute of Protocol Buffer message SignedEvent.
+	let signature: String
+}
```

Comment:
It's event redundant to traceLocation? Maybe this is an old file?

Reply:
Good point! Removed old file.